Source/CFRuntime.c

```diff
@@ -384,6 +384,25 @@ CFRetain (CFTypeRef cf)
   return cf;
 }
 
+CFTypeRef
+CFAutorelease (CFTypeRef cf)
+{
+  // FIXME: this is leaking
+  return cf;
+}
+
+void *
+_CFBridgingRelease (CFTypeRef cf)
+{
+  return objc_autoreleaseReturnValue(cf);
```

Comment:
Why are you using objc_autoreleaseReturnValue() instead of objc_release()? I remember some discussion, but can't remember the outcome.